src/plots/cartesian/graph_interact.js

```diff
@@ -2099,8 +2099,7 @@ fx.dragElement = function(options) {
         if(options.doneFn) options.doneFn(gd._dragged, numClicks);
 
         if(!gd._dragged) {
-            var e2 = document.createEvent('MouseEvents');
-            e2.initEvent('click', true, true);
+            var e2 = new MouseEvent('click', { 'button': e.button });
```

Comment:
Ah perfect! Seems to be working in Safari despite the `?` too so I guess Edge is the only unknown. :+1: